plugins/node/opentelemetry-plugin-ioredis/src/utils.ts

```diff
@@ -84,35 +84,40 @@ export const traceSendCommand = (
     this: ioredisTypes.Redis & IORedisPluginClientTypes,
     cmd?: IORedisCommand
   ) {
-    if (arguments.length >= 1 && typeof cmd === 'object') {
-      const span = tracer.startSpan(cmd.name, {
-        kind: SpanKind.CLIENT,
-        attributes: {
-          [AttributeNames.COMPONENT]: IORedisPlugin.COMPONENT,
-          [AttributeNames.DB_TYPE]: IORedisPlugin.DB_TYPE,
-          [AttributeNames.DB_STATEMENT]: dbStatementSerializer(
-            cmd.name,
-            cmd.args
-          ),
-        },
-      });
+    if (arguments.length < 1 || typeof cmd !== 'object') {
+      return original.apply(this, arguments);
+    }
+    if (tracer.getCurrentSpan() === undefined) {
```

Comment:
```suggestion
    // Do not trace if no parent span
    if (tracer.getCurrentSpan() === undefined) {
```

Just for ppl like me :)